corehq/apps/change_feed/document_types.py

```diff
@@ -38,6 +41,8 @@ def _get_primary_type(raw_doc_type):
         return WEB_USER
     elif raw_doc_type in ('Group', 'Group-Deleted'):
         return GROUP
+    elif raw_doc_type in app_doc_types().keys():
```

Comment:
nitpick: `raw_doc_type in app_doc_types().keys()` is less efficient than `raw_doc_type in app_doc_types()` (in Python 2) because the first one has to create a list by iterating over the dict and then look at each item in the list one by one to find a match. The second does a single hash lookup on the dict. In this case the dict is so small that it probably doesn't matter, so no need to change if you don't want to.

In general I don't like using `.keys()` because its return value is different in Python 2 vs Python 3, and it's almost always unnecessary anyway because iterating over a dict yields keys. If you really need a list of keys the idiomatic pattern is `list(the_dict)` in both Python 2 and 3.